src/Resources/config/doctrine/BasePage.orm.xml

```diff
@@ -22,6 +22,7 @@
         <field name="templateCode" type="string" column="template" nullable="false"/>
         <field name="createdAt" type="datetime" column="created_at"/>
         <field name="updatedAt" type="datetime" column="updated_at"/>
+        <field name="targetRedirectType" type="integer" column="target_redirect_type" nullable="true" default="302"/>
```

Comment:
This is a major change, because you need to run a database update

Reply:
Related: https://github.com/sonata-project/SonataPageBundle/pull/825#discussion_r121701047

Reply:
Ok, i'll create a new PR for master.